### PR TITLE
improved defaults and improved flow demonstration

### PR DESCRIPTION
**Pull request by ngupta23**
## Related Issuse or bug
  - Improved the defaults in create_model for time series.

Fixes: 
  - Set default trend to "add" (most models have None). Rationale: If trend is not present, "slope" equivalent in model will be small 
  - If seasonality is detected, Set default seasonal model to multiplicative. Rationale: If seasonality is additive, then the multiplicative component will be small

## Type of change

Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Unit test pass

## Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by Yard1 — CHANGES_REQUESTED** [review]
Just small nitpicks

File: pycaret/containers/models/time_series.py [hunk 1]
```diff
@@ -515,17 +542,30 @@ def __init__(self, globals_dict: dict) -> None:
...
-            "error": ["add", "mul"],
-            "trend": ["add", "mul", None],
-            # "damped_trend": [True, False],
-            "seasonal": ["add", "mul", None],
-            "sp": [sp],
-        }
-        tune_distributions = {}
+        if seasonality_present:
+            tune_grid = {
+                "error": ["add", "mul"],
```
[the remark this answers is not in the document]
Comment (ngupta23) [reply pycaret/containers/models/time_series.py:553]: Plus I prefer it to be explicit so it is easier to read :)

**Review by TremaMiguel — APPROVED** [review]

**Comment by TremaMiguel** [comment]
Just one thing, next meeting I will like to discuss if there's a better way to test the tuning methods, currently the CI takes more than an hour and most of the time testing the time_series module. Here is an image of `test_tune_model_random` durations. 

test_random

**Review by Yard1 — APPROVED** [review]

File: pycaret/containers/models/time_series.py [hunk 2]
```diff
@@ -454,37 +457,61 @@ def __init__(self, globals_dict: dict) -> None:
         sp = globals_dict.get("seasonal_period")
         sp = sp if sp is not None else 1
 
-        args = {"sp": sp, "seasonal": "add"} if seasonality_present else {}
+        # TODO: Check if there is a formal test for type of seasonality
+        args = {"sp": sp, "seasonal": "mul"} if seasonality_present else {}
+        # Add irrespective of whether seasonality is present or not
+        args.update({"trend": "add"})
```
[the remark this answers is not in the document]
Comment (ngupta23) [reply pycaret/containers/models/time_series.py]: Ok, I have updated it now.

**Pull request merged**